apps/static/dance/ai/model/cached-spacy-palette-map.json

```diff
@@ -0,0 +1,67 @@
+{
+  "emojiAssociations": {
+    "poopy": {
+      "association": [0.11, -0.16, 0.11, 0.14, 0.03, -0.04, -0.07, 0.12]
+    },
+    "romantic": {
+      "association": [0.28, 0.04, 0.08, 0.16, 0.22, 0.27, 0.3, 0.29]
+    },
+    "party": {
+      "association": [0.01, 0.24, 0.12, 0.04, 0.03, 0.15, 0.02, 0.03]
+    },
+    "silly": {
+      "association": [0.43, -0.14, 0.06, 0.21, 0.23, -0.11, 0.09, 0.14]
+    },
+    "sparkle": {
+      "association": [0.46, 0.22, 0.44, 0.46, 0.09, 0.41, 0.51, 0.52]
+    },
+    "happy": {
+      "association": [0.44, -0.0, 0.12, 0.22, 0.01, 0.02, 0.13, 0.36]
+    },
+    "magic": {"association": [0.34, 0.11, 0.23, 0.36, 0.1, 0.08, 0.24, 0.2]},
+    "spooky": {
+      "association": [0.46, -0.11, 0.07, 0.27, 0.3, 0.04, 0.13, 0.19]
+    },
+    "cute": {"association": [0.58, -0.11, 0.15, 0.42, 0.2, 0.08, 0.36, 0.31]},
+    "funky": {"association": [0.24, 0.15, 0.1, 0.13, 0.69, 0.08, 0.27, 0.15]},
+    "wavy": {"association": [0.26, 0.18, 0.24, 0.28, 0.04, 0.25, 0.31, 0.22]},
+    "lights": {
+      "association": [0.28, 0.39, 0.26, 0.42, -0.02, 0.33, 0.34, 0.3]
+    },
+    "rainbow": {
+      "association": [0.36, -0.05, 0.33, 1.0, 0.12, 0.23, 0.35, 0.34]
+    },
+    "robot": {
+      "association": [0.18, 0.31, 0.14, 0.08, 0.05, 0.01, 0.07, -0.04]
+    },
+    "chaotic": {
+      "association": [0.24, 0.12, 0.06, 0.15, 0.22, 0.2, 0.06, 0.17]
+    },
+    "disco": {"association": [0.27, 0.42, 0.25, 0.13, 0.5, 0.31, 0.35, 0.17]},
+    "zen": {"association": [0.06, 0.05, -0.04, -0.03, 0.12, 0.11, 0.1, 0.01]},
+    "fast": {"association": [0.37, 0.25, 0.23, 0.06, 0.06, 0.12, 0.11, 0.25]},
+    "evil": {
+      "association": [0.09, -0.04, 0.05, 0.15, 0.08, 0.01, -0.05, 0.01]
+    },
+    "cold": {"association": [0.55, 0.05, 0.46, 0.25, 0.03, 0.27, 0.09, 0.69]},
+    "cosmic": {"association": [0.19, 0.2, 0.14, 0.3, 0.15, 0.21, 0.09, 0.12]},
+    "sad": {
+      "association": [0.27, -0.2, -0.04, 0.18, 0.12, -0.1, -0.04, 0.12]
+    },
+    "black-and-white": {
+      "association": [0.08, 0.4, 0.2, 0.14, 0.12, 0.36, 0.33, 0.08]
+    },
+    "warm": {"association": [0.68, -0.02, 0.37, 0.34, 0.02, 0.33, 0.25, 1.0]},
+    "cool": {"association": [1.0, 0.06, 0.38, 0.36, 0.2, 0.22, 0.35, 0.68]}
+  },
+  "output": [

```

Comment:
Looks like this list is missing "black and white" and "light" from the standalone level.

Reply:
Those were deleted due to the current HoC not having render options for those two effects

Reply:
Similar to above comment, tracking regenerating to support these missing options later here: https://trello.com/c/856OLXRA